static/src/js/components/GranuleResults/GranuleResultsItem.jsx

```diff
@@ -182,6 +186,7 @@ const GranuleResultsItem = forwardRef(({
       ref={ref}
       role="button"
       tabIndex={0}
+      aria-label={itemTitle.title}
```

Comment:
Were you not able to select this element using a aria-role. I would think with `role="button"` it should be available without adding a label. Not a big deal if thats not the case. Just curious.